conf/machine/include/imx-base.inc

```diff
@@ -298,7 +298,7 @@ IMX_DEFAULT_KERNEL_use-mainline-bsp = "linux-fslc"
 PREFERRED_PROVIDER_virtual/kernel ??= "${IMX_DEFAULT_KERNEL}"
 
 SOC_DEFAULT_IMAGE_FSTYPES = "wic.bmap wic.gz"
-SOC_DEFAULT_IMAGE_FSTYPES_mxs = "uboot-mxsboot-sdcard wic.gz"
+SOC_DEFAULT_IMAGE_FSTYPES_mxs = "uboot-mxsboot-sdcard wic.bmap wic.gz"
```

Comment:
Please do the same change to the other default. So other SoC does the same.

Reply:
the other default is already creating a wic.bmap ...

Reply:
possibly for specific machines but not defined here. See the line above of your change.

Reply:
Sorry I am not understanding what you're saying
before my changes the two lines read as
```
SOC_DEFAULT_IMAGE_FSTYPES = "wic.bmap wic.gz"
SOC_DEFAULT_IMAGE_FSTYPES_mxs = "uboot-mxsboot-sdcard wic.gz"
```
so the problem here is for mxs the wic.bmap is not generated.
for the rest of the architectures is already there

After my changes:
```
SOC_DEFAULT_IMAGE_FSTYPES = "wic.bmap wic.gz"
SOC_DEFAULT_IMAGE_FSTYPES_mxs = "uboot-mxsboot-sdcard wic.bmap wic.gz"
```
I simply added the wic.bmap to SOC_DEFAULT_IMAGE_FSTYPES_mxs

Reply:
I am very sorry, I need to take a coffee ... you're right.

Reply:
Don't worry in the current situation were all on the same boat :-)